assets/src/application/ui/layout/entityList/EntityList/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { __ } from '@wordpress/i18n';
+import { without } from 'ramda';
+import classNames from 'classnames';
+
+import { Entity } from '@edtrServices/apollo/types';
+import EmptyState from '@appDisplay/EmptyState';
+import './style.scss';
+
+interface EntityListProps {
+	entities: Entity[];
+	EntityGridView: React.ReactNode;
+	EntityListView: React.ReactNode;
+	htmlClass: string;
+	view: string;
```

Comment:
`view` can be a union (`'grid' | 'list'`) or an enum.

Reply:
thx, done